### PR TITLE
Configure button binding done for the shooter code

### PR DESCRIPTION
**Pull request by adamyapathak**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -14,7 +14,19 @@
 public final class Constants {
 
     public static final class ShooterSpeed{
-        public static final double kShooterSpeed = 0.1;
+        public static final double kShooterSpeed = 0.5;
```
Comment (kjr-code) [inline src/main/java/frc/robot/Constants.java:17]: These are good data points to have in the `Constants.java` file; but we don't need to create an individual class for each one. We can group the constants in a few different ways -- I'd suggest grouping them by the purpose they serve. 

For instance, we can just have, say,

`public static final class ShooterConstants{}`
`public static final class IntakeConstants{}`

and then put all of the data dealing with the Shooter in `ShooterConstants` and all of the intake data in `IntakeConstants`

This saves us from having to make so many extra classes. It helps with maintenance, and is more memory efficient.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -25,13 +26,16 @@ public RobotContainer() {
     configureBindings();
 
     m_shoot.setDefaultCommand(m_shoot.stopMotorCommand());
```
Comment (kjr-code) [inline src/main/java/frc/robot/RobotContainer.java:28]: I see what you're going for here, and this is fine for now. There is, IMO, a slightly better way to recreate this behavior (stopping the motor when it's not in use), but the explanation is a bit wordy so we can go over that on Saturday.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
+import static frc.robot.Constants.IntakeSpeed.*;
 
-        // Stop the mechanism once the note is in
+public class Intake extends SubsystemBase{
 
-        // Limit switch or beam breaker to check if the note is in so the intake can communicate with the shooter?
+    CANSparkMax intakeMotor;
 
-        // Use pid to run the motors on different speeds?
+    public Intake(){
```
Comment (kjr-code) [inline src/main/java/frc/robot/subsystems/Intake.java:16]: OK I SEE YOU ADAMYA, WE'RE COOKING RN (good job on placing the variable instantiation in the constructor!) 👍

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 4]
```diff
@@ -1,24 +1,36 @@
...
+        intakeMotor = new CANSparkMax(kIntakeMotorID, MotorType.kBrushless);
+    }
 
-        System.out.print("Return something");
+    @Override
+    public void periodic() {};
 
-        return null;
+    public Command takeIn() {
```
Comment (kjr-code) [inline src/main/java/frc/robot/subsystems/Intake.java:23]: This should be fine; but if the Driver Station/Robot/VSCode yells at you for it, let me know. We might have to adjust the syntax a little bit (lambda functions are weird).

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 5]
```diff
@@ -1,24 +1,36 @@
...
+        return run(() -> {
+            intakeMotor.set(kIntakeSpeed);
+        });
+    }
 
+    public Command stopMotorCommand() {
```
Comment (kjr-code) [inline src/main/java/frc/robot/subsystems/Intake.java:29]: This is good (I remember us going over this bit a couple of meetings ago). Keep this. We'll use it later.

**Review by kjr-code — COMMENTED** [review]
Made some suggestions that should be fairly quick to implement. Give it a look and let me know what you think.

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (adamyapathak) [reply src/main/java/frc/robot/Constants.java:17]: 👍

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (adamyapathak) [reply src/main/java/frc/robot/RobotContainer.java:28]: 👍

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 3 above)
Comment (adamyapathak) [reply src/main/java/frc/robot/subsystems/Intake.java:16]: 👍

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 4 above)
Comment (adamyapathak) [reply src/main/java/frc/robot/subsystems/Intake.java:23]: 👍

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 5 above)
Comment (adamyapathak) [reply src/main/java/frc/robot/subsystems/Intake.java:29]: 👍

**Comment by adamyapathak** [comment]
👍

**Comment by adamyapathak** [comment]
I made the changes in constants.java

File: src/main/java/frc/robot/RobotContainer.java [hunk 6]
```diff
@@ -4,21 +4,16 @@
 
 package frc.robot;
 
-// import java.lang.reflect.InaccessibleObjectException;
-
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.*;
```
Comment (kjr-code) [inline src/main/java/frc/robot/RobotContainer.java:7]: Adam is going to make a comment about unused imports lol

**Review by kjr-code — APPROVED** [review]
LGTM, might make a few minor changes here and there but for now, 👍
